databricks-dbutils-scala/src/main/scala/com/databricks/sdk/scala/dbutils/SdkDBUtilsImpl.scala

```diff
@@ -20,26 +20,36 @@ private object SdkDbfsUtilsImpl {
 /** Help is a no-op in the SDK version of DBUtils. */
 trait NoHelp extends WithHelpMethods {
   override def help(): Unit = {}
+
   override def help(moduleOrMethod: String): Unit = {}
 }
 
 class SdkDBUtilsImpl(config: DatabricksConfig) extends DBUtils with NoHelp {
   private val client = new WorkspaceClient(config)
+
   def this() = this(new DatabricksConfig())
 
-  override def widgets: WidgetsUtils = SdkDbfsUtilsImpl.unsupportedField("widgets")
-  override def meta: MetaUtils = SdkDbfsUtilsImpl.unsupportedField("meta")
+  override def widgets: WidgetsUtils = SdkDBUtilsImpl.unsupportedField("widgets")
+
```

Comment:
nit: the newlines are unnecessary diff. can we keep the code blocks as intended by previous author?